src/main/java/org/broadinstitute/consent/http/resources/DataAccessRequestResource.java

```diff
@@ -135,16 +135,20 @@ public Response createDataAccessRequest(
   @RolesAllowed(RESEARCHER)
   @Path("/v3")
   public Response createDataAccessRequestWithDAARestrictions(
-      @Auth AuthUser authUser, @Context UriInfo info, String dar) {
+      @Auth AuthUser authUser,
+      @Context Request request,
+      @Context UriInfo info, String dar) {
     try {
       User user = findUserByEmail(authUser.getEmail());
       DataAccessRequest payload = populateDarFromJsonString(user, dar);
       // DAA Enforcement
       datasetService.enforceDAARestrictions(user, payload.getDatasetIds());
-      DataAccessRequest newDar = dataAccessRequestService.createDataAccessRequest(user, payload);
+      DataAccessRequest newDar = dataAccessRequestService.createDataAccessRequest(user, payload, (ContainerRequest) request);
       sendNewDarCollectionMessage(newDar.getCollectionId());
       URI uri = info.getRequestUriBuilder().build();
       matchService.reprocessMatchesForPurpose(newDar.getReferenceId());
+      List<Dataset> datasets = datasetService.findDatasetsByIds(newDar.getDatasetIds());
+      ComplianceLogger.logDARSubmission(user, datasets, ((ContainerRequest) request), HttpStatusCodes.STATUS_CODE_CREATED);
```

Comment:
Drive by fix. This should have been added when we added the same code on lines 124-125.